civihr_default_theme/scss/drupal/_form.scss

```diff
@@ -145,18 +161,20 @@
     .date-padding {
       position: relative;
 
-      &:before {
+      &::before {
         content: '\f073';
         font-family: FontAwesome;
         position: absolute;
         right: -20px;
         top: 7px;
       }
 
+      /* stylelint-disable selector-max-compound-selectors */
```

Comment:
can't we refactor the style instead

Reply:
I have refactored the styles which could be, but where it could not be done because of the required structure, linting has been disabled. This is similar.